pkgs/applications/window-managers/notion/default.nix

```diff
@@ -1,39 +1,32 @@
 {
-  enableXft ? true, libXft ? null, patches ? [], stdenv, lua, gettext, pkgconfig, xlibsWrapper, libXinerama, libXrandr, libX11,
-  xterm, xmessage, makeWrapper, fetchFromGitHub, mandoc, which
+  enableXft ? true, libXft ? null, stdenv, lua, gettext, readline, pkgconfig, xlibsWrapper, libXinerama, libXrandr, libX11,
+  xterm, xmessage, makeWrapper, fetchFromGitHub, mandoc, which, groff
 }:
 
 assert enableXft -> libXft != null;
 
 let
   pname = "notion";
-  version = "3-2017050501";
-  inherit patches;
+  version = "4.0.git";
 in
 stdenv.mkDerivation {
   name = "${pname}-${version}";
   meta = with stdenv.lib; {
-    description = "Tiling tabbed window manager, follow-on to the ion window manager";
-    homepage = http://notion.sourceforge.net;
+    description = "Tiling tabbed window manager";
+    homepage = https://notionwm.net;
     platforms = platforms.linux;
-    license   = licenses.notion_lgpl;
+    license   = licenses.lgpl21;
     maintainers = with maintainers; [jfb];
```

Comment:
Would you like to be a maintainer? (That'd be really cool, since you're the main author of Notion :))

Reply:
Added myself to the list ;)